experimental/sample.go

```diff
@@ -0,0 +1,109 @@
+package experimental
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+)
+
+//----------------------------------------------------------------------------------
+// PLUGIN HOST
+//----------------------------------------------------------------------------------
+
+// MyHost singelton host service
+type MyHost struct{}
+
+// CheckExeHealth check if the plugin host is running
```

Comment:
I'm not 100% sure about the intention of this function, but this comment is more descriptive at least:
```suggestion
// CheckExeHealth returns a backend.CheckHealthResult.
```